router/core/graphql_prehandler.go

```diff
@@ -242,10 +249,17 @@ func (h *PreHandler) Handler(next http.Handler) http.Handler {
 
 				writeOperationError(r, w, requestLogger, err)
 				h.releaseBodyReadBuffer(buf)
+				operationReadSpan.End()
 				return
 			}
 		}
 
+		operationReadSpan.SetAttributes(
+			otel.WgGraphQLRequestSizeBytes.Int(len(body)),
+			otel.WgMultipartFilesCount.Int(len(files)),
```

Comment:
Please check if there is already a known otel attribute for that. If not we can invent it but we should keep it free from `wg` prefix because it is HTTP related. Maybe `http.request.files_count`